ledger/participant-state/kvutils/src/main/protobuf/com/daml/ledger/participant/state/kvutils/daml_kvutils.proto

```diff
@@ -151,6 +159,8 @@ message DamlTransactionEntry {
 
   // The pre-computed transaction blinding information.
   DamlTransactionBlindingInfo blinding_info = 7;
+
```

Comment:
If there's a one-to-one correspondence between the divulged contracts listed here and the entries in `blinding_info`, why not simply add a new field to `DivulgenceEntry` containing the contract instance? That way you the relation between contract instances and divulgence entries would be explicit, making their consumption easier.

Reply:
good point, will update

Reply:
Updated, albeit with a bit more code impact